### DISCUSSION TITLE
Forms

### DISCUSSION THREAD

**Discussion by rafael-legyo** [vercel/next.js#18392] [Ideas]
Vercel could add form functionality exactly as it did with Netlify. I particularly migrated the site there because of that.

**Comment by JonCognioDigital** [comment]
Maybe I've musunderstood your question but can't you use any forms library?

i started out using Formik but ditched it in favour of use-react-form

**Reply by rafael-legyo** [reply]
I say to make it work, to store data, etc.
Just add a tag

`<form ... data-netlify="true">`

**Reply by renet** [reply]
I think what Rafael is talking about is, that Netlify offers a (free) forms functionality that lets you submit a form and Netlify collects the submits and forwards them to wherever you want. He suggests that Vercel adds such a functionality to its core features.

@rafael-legyo, you can still use the free plans of a thirds party form service with Vercel. I quickly found 2 guides from Vercel that help you with two different services:
- [Formspree](https://vercel.com/guides/deploying-react-forms-using-formspree-with-vercel) (50 submits / month for free)
- [Formcarry](https://vercel.com/guides/vue-js-html-forms-with-formcarry-vercel) (100 submits / month for free)
